depmapomics/config_dev.py

```diff
@@ -1,3 +1,5 @@
+version = '23Q4'
```

Comment:
This is already in constants.py, so no need to add them in here and config_prod again. We only need to make the variable in constants.py a default input for the function you added. I'll approve and fix that myself. Thanks!